app/facet_selector/facet_selector.controller.js

```diff
@@ -12,6 +12,8 @@
       $scope.transfers = Transfer;
       $scope.tags = Tag;
 
+      $scope.date_regex = '\\d\\d\\d\\d([-\/]\\d\\d)?([-\/]\\d\\d)?';
```

Comment:
Also allow 1 digit month/days? eg 2000-1-1  Do we want to support two digit years (99-12-31)?